gamma_bridge/tasks.py

```diff
@@ -0,0 +1,33 @@
+"""Celery tasks for working asynchronously."""
+import socket
+
+from celery.task import task
+
+from gamma_bridge import settings
+from gamma_bridge.exceptions import GammaConnectionError
+from gamma_bridge.storage_logger import StorageLogger
+from gamma_bridge.storage import GammaStorage
+
+
+@task(bind=True)
+def publish_event_to_gamma_task(self, params, event, sleep_interval):
```

Comment:
Lets rename it to the `publish_event_to_gamma `.

Reply:
This method was renamed.